Project1/Unit.h

```diff
@@ -108,4 +107,8 @@ class Unit : public Observable<ObserverDamage>, public Observable<ObserverExp> {
 	}
 	bool operator==(const Unit & unit) const;
 	bool operator!=(const Unit & unit) const;
-};
\ No newline at end of file
+};
+//TODO (Weston): Is this an ok place for this?
```

Comment:
I think these should go inside of classes, ie
`Unit::Ref`
`Unit::ConstRef`
`Unit::UniquePtr`